atom-client/src/test/java/org/motechproject/atomclient/it/AtomClientServiceBundleIT.java

```diff
@@ -0,0 +1,118 @@
+package org.motechproject.atomclient.it;
+
+import org.apache.http.HttpStatus;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.motechproject.atomclient.repository.FeedRecordDataService;
+import org.motechproject.atomclient.service.AtomClientConfigService;
+import org.motechproject.atomclient.service.AtomClientService;
+import org.motechproject.atomclient.service.FeedConfig;
+import org.motechproject.atomclient.service.FeedConfigs;
+import org.motechproject.mds.query.SqlQueryExecution;
+import org.motechproject.testing.osgi.BasePaxIT;
+import org.motechproject.testing.osgi.container.MotechNativeTestContainerFactory;
+import org.motechproject.testing.osgi.http.SimpleHttpServer;
+import org.ops4j.pax.exam.ExamFactory;
+import org.ops4j.pax.exam.junit.PaxExam;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerSuite;
+
+import javax.inject.Inject;
+import javax.jdo.Query;
+import java.util.Arrays;
+import java.util.HashSet;
+
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(PaxExam.class)
+@ExamReactorStrategy(PerSuite.class)
+@ExamFactory(MotechNativeTestContainerFactory.class)
+public class AtomClientServiceBundleIT extends BasePaxIT {
+
+    @Inject
+    private AtomClientService atomClientService;
+    @Inject
+    private AtomClientConfigService configService;
+    @Inject
+    private FeedRecordDataService feedRecordDataService;
+
+
+    @Before
+    public void setupDatabase() {
+        SqlQueryExecution sqe = new SqlQueryExecution() {
+
+            @Override
+            public String getSqlQuery() {
+                return "DELETE FROM atomclient_feed_record WHERE 1=1";
```

Comment:
Why are we clearing the db using SQL query in the test class? The database is cleared before ITs by Maven. We could leave it here to ease running tests via IDE, but then why not use data service to remove all elements?